About_Page.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>Sara Kuddah</title>
+    <link rel="stylesheet" href="css/styles.css">
+</head>
+<body>
```

Comment:
indent your code to make it easier to read 

here is a link that will help
https://www.granneman.com/webdev/coding/formatting-and-indenting-your-html